v2.0/delete.md

```diff
@@ -6,6 +6,8 @@ toc: false
 
 The `DELETE` [statement](sql-statements.html) deletes rows from a table.
 
+{{site.data.alerts.callout_danger}}If you delete a row that is referenced by a <a href="foreign-key.html">foreign key</a> and has an <a href="foreign-key.html#foreign-key-actions"><code>ON DELETE</code> action</a>, all of the dependent rows will also be deleted or updated.{{site.data.alerts.end}}
```

Comment:
nit: `a foreign key and has an ON DELETE action` > `a foreign key constraint with an ON DELETE action`

Reply:
Done